style.css

```diff
@@ -0,0 +1,698 @@
+body {
+  margin: 0;
+  overflow-x: hidden;
+  transition: all 0.3s;
+}
+
+.color1 {
+  color: #272a31 !important;
+}
+
+.color2 {
+  color: #ec5242 !important;
+}
+
+#error-popup {
+  position: fixed;
+  color: #fff;
+  font-size: 1em;
+  font-family: cocogoose-b, sans-serif;
+  margin: auto;
+  left: 0;
+  right: 0;
+  margin-top: 100px;
+  width: 400px;
+  height: 60px;
+  visibility: visible;
+  opacity: 1;
+  transition: all 3s;
+  border-radius: 4%;
+  text-align: center;
+  line-height: 60px;
+  z-index: 99;
+}
+
+.red {
+  background: red;
+}
+
+.green {
+  background: green;
+}
+
+.fade {
+  visibility: hidden !important;
+  opacity: 0 !important;
+}
+
+main {
+  background-image: url('./img/main.png');
+  height: auto;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.bold {
+  font-weight: bold !important;
+  font-family: cocogoose-b, sans-serif;
+  font-size: 1.1em;
+}
+
+#menu-top {
```

Comment:
### [Optional]
Your code appears to be WET-(Write Every Time), Microverse encourage DRY-(Don't Repeat Yourself) code.

For example In your case; you repeated `display: flex;`   `flex-direction: column;`, and so on...

Could be combined like this:

#menu-top,  #welcome.....{
  display: flex;
  flex-direction: column;
}

Let's improve our styling by using [Combine Elements](https://speckyboy.com/good-bad-css-practices)

Reply:
I will work on it and improve that 
Thank you for that